packages/dm-core-plugins/src/form/widgets/TextareaWidget.tsx

```diff
@@ -14,13 +14,18 @@ const TextareaWidget = (props: TWidget) => {
 
   return (
     // @ts-ignore
-    <TextField
-      {...props}
-      multiline={true}
-      rows={5}
-      onChange={onChangeHandler}
-      label={label}
-    />
+    <>
+      <Typography className={'self-center pb-2'} bold={true}>
+        {label}
+      </Typography>
```

Comment:
If you wrap this in a `<label htmlFor={label}>` it will tell the screen-readers about which field it belongs to. ({label} because thats whats used as id in the `TextField` component)

Reply:
<img width="621" alt="image" src="https://github.com/equinor/dm-core-packages/assets/37016135/13bca7bc-8d78-451d-9f15-9f09c91d0d97">
Like this?

Reply:
Is this correct, also setting the id to be the label?

Reply:
Would not this be more semantically correct, using id instead of label? 
<img width="688" alt="image" src="https://github.com/equinor/dm-core-packages/assets/37016135/d8c908ca-ac70-454b-81dc-019902a91c37">

Reply:
When digging through the dom it looks like EDS applies the label as the id of the textarea. Thats why I suggest using the label in the `htmlFor`. If the id you pass on the `TextField` component gets passed down to the `textfield` tag, you may use that instead.